.gitlab-ci.yml

```diff
@@ -48,16 +48,15 @@ deploy_to_reliability_env:
 tracer-base-image:
   extends: .ci_authenticated_job
   rules:
-    - if: '$CI_COMMIT_BRANCH == "master"'
+    - if: '$CI_PIPELINE_SOURCE == "schedule"'
```

Comment:
Doesn't it make more sense to upload from master rather than on a trigger? Why the change?

Reply:
I am not sure when the trigger is launched.....When the GitLab trigger is launched, CircleCI build is finished??
I am worried that things like this could happen: https://github.com/DataDog/dd-trace-js/issues/2677#issuecomment-1426198898

Reply:
@pierotibou Since we're fetching artifact from circleci last job. We currently don't have any means to wait for circle ci job to finish - to fetch artifact.

At this point having a schedule and uploading last master circleci job periodically makes this a little bit more predictable and will cause somewhat lower traffic overall.